package.json

```diff
@@ -35,7 +35,7 @@
     "material-ui-search-bar": "1.0.0-beta.13",
     "md5": "2.2.1",
     "moment": "2.24.0",
-    "npm": "6.13.4",
+    "npm": "^6.13.7",
```

Comment:
This shouldn't be the part of the `package.json`. I am not sure why is it even present. Please remove it.

Reply:
@akshatnitd so need to remove npm or stick with the old version

Reply:
`npm` should not be a dependency

Reply:
okay remove it